frameworks/solid/src/benchmark.jsx

```diff
@@ -164,15 +164,15 @@ const Benchmark = () => {
         await warmup(createMany, 1000, 5)
         const createRes = await createMany(1000)
         results.create = createRes.time.toFixed(2);
-
+        
         await createMany(1000);
         await warmup(updateMany, 1000, 5)
         await createMany(1000);
         const updateRes = await updateMany(1000, 0);
         results.update = updateRes.time.toFixed(2);
 
         await createMany(1000);
-        await warmup(updateMany, [1000, 10], 5);
```

Comment:
all `updateMany` tests are run with 5 warmup rounds across all frameworks, this will include solid. Please revert.

Reply:
You were passing in an array so it wasn't correct - the next line is accurate.

Reply:
That's because this is the skip test, it needs a second argument. That array turns into arguments to call when the warmup runs.